lib/Phpactor.php

```diff
@@ -51,7 +51,13 @@ public static function boot(InputInterface $input, string $vendorDir): PhpactorC
 
         $config = $loader->load();
         $config[CoreExtension::COMMAND] = $input->getFirstArgument();
-        $config[FilePathResolverExtension::PARAM_APPLICATION_ROOT] = realpath(__DIR__ . '/..');
+
+        if (is_dir(realpath(__DIR__ . '../vendor'))) {
+            $config[FilePathResolverExtension::PARAM_APPLICATION_ROOT] = realpath(__DIR__ . '/..');
+        } else {
+            $config[FilePathResolverExtension::PARAM_APPLICATION_ROOT] = realpath(__DIR__ . '/../../../..');
+        }
```

Comment:
can we extract to a method and remove the `else`?

maybe even something like the following?

```
private function resolveApplicationRoot() 
{
   $paths = [ __DIR__ . '/../vendor', __DIR__ .'/../../../..' ];
   foreach ($paths as $path) { if (is_dir($path)) { return $path; } }
   throw new RuntimeException(sprintf('Could not resolve application root, tried "%s"', implode('", "', $paths)));
   }
}
```

Reply:
sure ! (I kept the call to `realpath`)